### DISCUSSION TITLE
Issue: Unable to Accept Assignment on GitHub Classroom

### DISCUSSION THREAD

**Discussion by LUISMK2** [community/community#140295] [GitHub Education] [answered]
### Hi everyone,

Hi GitHub community,

I'm encountering an issue with accepting an assignment in GitHub Classroom. When I try to accept the assignment, I receive the following error message:

"We are unable to access the starter code for this assignment. Please contact your Classroom admin."

This issue is blocking my progress in the learning path for GitHub, and I'm unsure how to resolve it. I've already verified my account and checked permissions. Has anyone experienced this before, or do you have suggestions on how to fix it?

Thanks in advance for your help!

**Comment by ShreyaSev** [comment]
Facing the same issue

**Reply by LUISMK2** [reply]
Kindly let me know once you have found a solution.

**Reply by vinod820** [reply]
I am also facing the same issue

**Reply by VinaySiddha** [reply]
i am also facing the issue We are unable to access the starter code for this assignment. Please contact your Classroom admin.

**Comment by fidelp27** [comment]
Someone solve this error?

**Comment by SCLiao47** [comment]
Same issue with "[Intro to GitHub Flow](https://education.github.com/experiences/series_intro_github_flow)"

**Reply by imenbkr** [reply]
I'm having the same issue with Github Flow

**Comment by pranusri1903** [comment]
Same issue with "[Intro to GitHub Flow]"
Contact if found a solution.

**Comment by codeiox** [comment]
I also have the same issue and am unsure how to solve it.

**Comment by alizeeshan-07** [comment]
Facing the same issue.

**Comment by VinaySiddha** [comment]
We are unable to access the starter code for this assignment. Please contact your Classroom admin.

**Comment by EdTosoy** [comment]
facing the same issue
<img width="715" alt="image" src="https://github.com/user-attachments/assets/ee360763-2b7e-4938-b0d0-afac18705cc8">

**Comment by Tonzinio** [comment]
Also facing the same issue

**Comment by LUISMK2** [comment, marked as answer]
Hi everyone,

I wanted to share a solution I found for the issue where you get the error: "We are unable to access the starter code for this assignment. Please contact your Classroom admin."

After reaching out to GitHub support and inquiring further, I was directed to a new version of the GitHub Classroom experience. You can access it here:

[GitHub Experience - Introduction to GitHub](https://education.github.com/experiences/intro_to_github)

This link will take you directly to the updated skills course where you can continue with the flow.

I hope this helps anyone else facing the same issue!

Best Regards,
LUISMK2
(Rishabh Singh)

**Reply by fidelp27** [reply]
Thank you
